src/@next/Select/Select.tsx

```diff
@@ -114,6 +114,14 @@ export const Select = ({
     setMenuOptions(newState);
   };
 
+  const handOnBlur = () => {
+    setTimeout(() => {
+      setPopoverActive(false);
+    }, 101);
```

Comment:
We have to set a timeout of more than 100 because of https://github.com/glints-dev/glints-aries/blob/0655a3bc8579d129461ad4fafd0f03788ef6e94e/src/@next/Select/components/SearchableSelectInput/SearchableSelectInput.tsx#L134

Reply:
This is a hacky solution but I guess it prevents a re-write

Reply:
@yongchenglow thanks for submitting this.
upon checking, this solution still has the same issues found here: https://github.com/glints-dev/glints-aries/pull/1000#pullrequestreview-1783309335

Reply:
@yongchenglow it's still having some issues on other types like the searchable single select:
![Kapture 2023-12-27 at 10 51 12](https://github.com/glints-dev/glints-aries/assets/7514754/5a0ad594-7c0b-4d0b-b93b-17b4a1da5686)

it seems it could still lead to some issues since onBlur event goes first then the onClick. I've read from [here](https://stackoverflow.com/questions/17769005/onclick-and-onblur-ordering-issue) we can try replacing our onClick to onMouseDown to let the browser reorder the event handlers. Maybe we can try this?

Reply:
Will be closing this MR. We will adjust to use our custom component combo box which are not facing these issues. We are facing several other issues as well which is heavily affecting our forms